tests/devices/default_tensor/test_tensor_var.py

```diff
@@ -362,6 +361,8 @@ def test_PauliZ_hadamard_PauliY(self, theta, phi, varphi, dev, tol):
         assert np.allclose(calculated_val, reference_val, tol)
 
 
+# This test is only for the new opmath since there is an error
+# in the tape computation with `default.qubit`, that we use as reference.
```

Comment:
Wait what? What is the error? Is it known and/or being tracked?

Reply:
This reproduces the error:

```
qml.operation.disable_new_opmath()

ops = [
    qml.PauliX(wires=[0]),
    qml.RX(theta, wires=[0]),
    qml.CSWAP(wires=[7, 0, 5]),
    qml.RX(phi, wires=[1]),
    qml.CNOT(wires=[3, 4]),
    qml.DoubleExcitation(phi, wires=[1, 2, 3, 4]),
    qml.CZ(wires=[4, 5]),
    qml.Hadamard(wires=[4]),
    qml.CCZ(wires=[0, 1, 2]),
    qml.CSWAP(wires=[2, 3, 4]),
    qml.QFT(wires=[0, 1, 2]),
    qml.CNOT(wires=[2, 4]),
    qml.Toffoli(wires=[0, 1, 2]),
    qml.DoubleExcitation(phi, wires=[0, 1, 3, 4]),
]

meas = [
    qml.var(qml.PauliY(wires=[2])),
    qml.var(qml.Hamiltonian([1, 5, 6], [qml.Z(6), qml.X(0), qml.Hadamard(4)])),
    qml.var(
        qml.Hamiltonian(
            [4, 5, 7],
            [
                qml.Z(6) @ qml.Y(4),
                qml.X(0),
                qml.Hadamard(7),
            ],
        )
    ),
]

tape = qml.tape.QuantumScript(ops=ops, measurements=meas)

dev = qml.device("default.qubit", max_workers=1)
program, _ = dev.preprocess()
tapes, transf_fn = program([tape])
results = dev.execute(tapes)
transf_fn(results)
```

I am wondering how much we should be worried, since this only happens with opmath disabled

Reply:
Mhm, not sure. But would be good to open an issue for this and resolve it there, as it is an independent discussion, I'd say :) Then you can link that issue in the default tensor test, and the pytest marker can be removed once the issue is resolved :)

Reply:
> would be good to open an issue for this

I agree, I'll do that tomorrow (maybe it is better to discuss this on slack first ?)

> you can link that issue in the default tensor test, and the pytest marker can be removed once the issue is resolved

I agree on this as well

> resolve it there

I'm not sure I understand what you mean. I don't think this PR should be blocked by this 'potential' issue, since this is totally independent of `default.tensor` (I encountered this only because I am using `default.qubit` as a reference). But if we decide that we want to resolve this, I'll certainly remove the pytest marker in this specific example

Reply:
By 

> resolve it there

I meant exactly that we don't need to hold back with this PR because of the issue :)